hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioOutboundPipeline.java

```diff
@@ -272,6 +280,7 @@ private void flushToSocket() throws IOException {
         lastWriteTime = currentTimeMillis();
         int written = socketChannel.write(sendBuffer);
         bytesWritten.inc(written);
+        bytesWrittenPerProtocol.inc(channel.protocolType, written);
```

Comment:
Not directly related, but still. The next line after this one is:
```
//System.out.println(channel+" bytes written:"+written);
```

Looks like a debugging garbage. Does it make sense to clean it as a part of this PR?

Reply:
In other places around here, there were comments saying that please do not delete these commented lines because it's useful for debugging. I assume it's the same for this line, so I'd rather leave it as is and let code owner of this part (it's Peter I guess) to handle it.

Reply:
OK, still it looks like some garbage code. I was surprised to see it.